src/components/BlogPreviewProfile.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import Container from "@material-ui/core/Container";
+import Typography from "@material-ui/core/Typography";
+import Divider from "@material-ui/core/Divider";
+import ButtonBase from "@material-ui/core/ButtonBase";
+import { useHistory } from "react-router-dom";
+
+const dataDomain =
+  window.location.hostname === "localhost" ? "localhost" : "skyblog.hns"; //domain for post route
+
+//BlogPreviewProfile component is a preview of blog posts for display in a feed
+export default function BlogPreviewProfile(props) {
+  const { post, id } = props; //post data and post id
+  const history = useHistory(); //react router hook
+
+  //handle date conversion for display
+  const getDateDisplay = (ts) => {
+    const d = new Date(ts);
+    return d.toDateString();
+  };
+
+  //handle routing to blog view page
+  const postRoute = () => {
```

Comment:
follow up: Will this be used in other places in the app? if so, we should refactor.

At a minimum things like `/feed-dac.hns/` and `/posts/page_0.json/` should be constants it seems.